documentation/source/task_bfms.rst

```diff
@@ -0,0 +1,192 @@
+###############
+Task-Based BFMs
+###############
+
+Overview
+========
+
+Cocotb supports a task-based bus functional model (BFM) interface between 
+Python, Verilog, and SystemVerilog (VHDL is TBD). 
+
+BFM Implementation (Python)
+===========================
+
+The Python aspect of a BFM is captured as a Python class with the @cocotb.bfm 
```

Comment:
This will both mark up and link the decorator name:
```suggestion
The Python aspect of a BFM is captured as a Python class with the :class:`cocotb.bfm`
```

Reply:
Ah, it's still not hyperlinked because the whole module should be getting a section in ``library_reference.rst``.
(Also, you are referencing the same decorator a few lines down, please add the same markup there as well.)